source/functions/BEMIO/readWAMIT.m

```diff
@@ -89,10 +89,10 @@
         hydro(F).Khs(3:5,3,hydro(F).Nb) = tmp{1};
         tmp = textscan(raw{n+2}(find(raw{n+2}==':')+1:end),'%f');
         hydro(F).Khs(4,4:6,hydro(F).Nb) = tmp{1};
-        hydro(F).Khs(4:6,4,hydro(F).Nb) = tmp{1};
+        hydro(F).Khs(4:5,4,hydro(F).Nb) = tmp{1}(1:2);
         tmp = textscan(raw{n+3}(find(raw{n+3}==':')+1:end),'%f');
         hydro(F).Khs(5,5:6,hydro(F).Nb) = tmp{1};
-        hydro(F).Khs(5:6,5,hydro(F).Nb) = tmp{1};
+        % 6th row must be zeros
```

Comment:
@salhus Minor comment, can we delete this line. Without more context a user might not understand why and we do not provide greater description in our documentation. Therefore, requesting to either delete, or reference the AQWA or WAMIT manuals.

Reply:
Sounds good. I just made that commit.